src/graphql/resolvers/token-resolver.ts

```diff
@@ -7,8 +7,8 @@ import { GraphQLContext } from "../context-type";
 
 @ArgsType()
 class TokensQueryArgs {
-  @Field(() => String, { nullable: true })
-  tokenId?: string;
+  @Field(() => [String], { nullable: true })
+  tokenIds?: [string];
```

Comment:
Would be good to have a limit here, please take a look at [AddressResolver](https://github.com/capt-nemo429/ergo-graphql/blob/master/src/graphql/resolvers/address-resolver.ts#L23) for more info.

Reply:
Done, I've set max size to 20. Let me know if you think some other constant suits better.